css/admin.upgrade.css

```diff
@@ -118,7 +118,7 @@
 	padding: 4rem 2.5rem;
 	color: #fff;
 	background-size: cover;
-	background-image: url('https://wpengine.com/wp-content/uploads/2020/07/gcb_gen_pro_hero_bg_img.png');
+	background-image: url('../assets/images/gcb_gen_pro_hero_bg_img.png');
```

Comment:
Could you change the image file names to match the new file names?

They should be `kebab-case` instead of `snake_case`.

You'll know it worked when the images appear:

![images-show-here](https://user-images.githubusercontent.com/4063887/124159456-cca4cc80-da60-11eb-8f84-ec5a8dc2f6fa.png)